include/wally_psbt.h

```diff
@@ -535,12 +547,62 @@ WALLY_CORE_API int wally_psbt_set_global_tx(
     struct wally_psbt *psbt,
     const struct wally_tx *tx);
 
+/**
+ * Add a transaction input to PBST at a given position.
+ *
+ * :param psbt: The PSBT to add the input to.
+ * :param index: The zero-based index of the position to add the input at.
+ * :param flags: Flags controlling input insertion. Must be 0 or ``WALLY_PSBT_FLAG_NON_FINAL``.
+ * :param input: The transaction input to add.
+ */
+WALLY_CORE_API int wally_psbt_add_input_at(
+    struct wally_psbt *psbt,
+    uint32_t index,
+    uint32_t flags,
+    const struct wally_tx_input *input);
+
+/**
+ * Remove a transaction input from a PBST.
+ *
+ * :param psbt: The PSBT to remove the input from.
+ * :param index: The zero-based index of the input to remove.
+ */
+WALLY_CORE_API int wally_psbt_remove_input(
+    struct wally_psbt *psbt,
+    uint32_t index);
+
+/**
+ * Add a transaction output to PBST at a given position.
+ *
+ * :param psbt: The PSBT to add the output to.
+ * :param index: The zero-based index of the position to add the output at.
+ * :param flags: Flags controlling output insertion. Must be 0.
+ * :param output: The transaction output to add.
+ */
+WALLY_CORE_API int wally_psbt_add_output_at(
+    struct wally_psbt *psbt,
+    uint32_t index,
+    uint32_t flags,
+    const struct wally_tx_output *output);
+
+/**
+ * Remove a transaction output from a PBST.
+ *
+ * :param psbt: The PSBT to remove the output from.
+ * :param index: The zero-based index of the output to remove.
+ */
+WALLY_CORE_API int wally_psbt_remove_output(
+    struct wally_psbt *psbt,
+    uint32_t index);
+
 /**
  * Create a PSBT from its serialized bytes.
  *
  * :param bytes: Bytes to create the PSBT from.
  * :param len: Length of ``bytes`` in bytes.
```

Comment:
s/len/bytes_len/ so swig handle this correctly

Reply:
Updated, thanks!